server/.env-example

```diff
@@ -1,3 +0,0 @@
-DB_USER=user
-DB_HOST=localhost:27017
-DB_PASSWORD=password
```

Comment:
why did you push these changes?

Reply:
Sorry. I guess I have made something wrong, while configuring local MongoDB. Just wanted to add myself to list of contributors.